certified-connectors/tyntec-SMS/apiDefinition.swagger.json

```diff
@@ -235,6 +248,300 @@
+                  "x-ms-visibility": "internal"
+                },
+                "content": {
+                  "type": "object",
+                  "properties": {
+                    "contentType": {
+                      "type": "string",
+                      "description": "contentType",
+                      "title": "",
+                      "x-ms-visibility": "internal",
+                      "default": "text"
+                    },
+                    "text": {
+                      "type": "string",
+                      "description": "text"
+                    }
+                  },
+                  "description": "content",
+                  "required": [
+                    "contentType"
+                  ]
+                }
+              },
+              "required": [
+                "channel",
+                "content"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/conversations/v3/power-automate/webhooks/channels/sms/phone-numbers/{smsSender}": {
+      "x-ms-notification-content": {
+        "description": "Response",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "messageId": {
+              "type": "string",
+              "description": "messageId"
+            },
+            "channel": {
+              "type": "string",
+              "description": "channel"
+            },
+            "from": {
+              "type": "string",
+              "description": "from"
+            },
+            "to": {
+              "type": "string",
+              "description": "to"
+            },
+            "receivedAt": {
+              "type": "string",
+              "description": "receivedAt"
+            },
+            "content": {
+              "type": "object",
+              "properties": {
+                "contentType": {
+                  "type": "string",
+                  "description": "contentType"
+                },
+                "text": {
+                  "type": "string",
+                  "description": "text"
+                },
+                "file": {
+                  "type": "string",
+                  "description": "file"
+                }
+              },
+              "description": "content"
+            },
+            "event": {
+              "type": "string",
+              "description": "event"
+            },
+            "timestamp": {
+              "type": "string",
+              "description": "timestamp"
+            }
+          }
+        }
+      },
+      "put": {
+        "responses": {},
+        "operationId": "incomingV3",
+        "summary": "Incoming message",
+        "description": "Triggers on Incoming message",
+        "x-ms-trigger": "single",
+        "deprecated": false,
+        "x-ms-api-annotation": {
+          "family": "Incoming",
+          "revision": 2
+        },
+        "parameters": [
+          {
+            "name": "smsSender",
```

Comment:
@ridlees The 'x-ms-summary' property and the 'description' property are required. please add both to the parameter.

Reply:
Added, thanks @Amjed-Ayoub <3